http-server/src/main/java/io/strimzi/http/server/RestService.java

```diff
@@ -0,0 +1,15 @@
+package io.strimzi.http.server;
+
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+import io.vertx.ext.web.Router;
+import org.javatuples.Triplet;
+
+/**
+ * The RestService interface is used to identify modules that wish to expose a set of REST endpoints
+ * on the {@link AdminServer}. The service returns a plugin name, a mount
+ * point and a {@link io.vertx.ext.web.Router} containing the endpoints for the plugin.
+ */
+public interface RestService {
```

Comment:
To me this interface is named wrong. It's defining the responsibilities of a _module_, and the module will define REST endpoints/services, so a more natural name would be `RestModule`, imho. I know you're using the java Service API, but I don't think that's enough reason to require `Service` in the name.